redirects.js

```diff
@@ -19,6 +20,37 @@ const setup = (app) => {
   app.route(["/narratives/ncov", "/narratives/ncov/sit-rep"])
     .get((req, res) => res.redirect('/#ncov'));
 
+  /*
+   * Redirect to translations of narratives if the client has
+   * set language preference and the translation is available
+   *
+   * Currently only used for nCoV narratives and requires the
+   * language code to be the second to last part of the url.
+   */
+  app.route("/narratives/ncov/sit-rep/*")
+    .get(async (req, res, next) => {
+      const {source, prefixParts} = helpers.splitPrefixIntoParts(req.url);
+      const availableNarratives = await source.availableNarratives();
+      const availableLanguages = new Set(availableNarratives.map((narrative) => {
+        if (narrative.startsWith('ncov/sit-rep/')) {
+          const narrativeParts = narrative.split("/");
+          const language = narrativeParts[narrativeParts.length - 2];
+          if (language !== 'sit-rep') return language;
+        }
+        return null;
+      }));
+      acceptedLanguage.languages(['en'].concat([...availableLanguages]));
+      const languageChoice = acceptedLanguage.get(req.headers['accept-language']);
```

Comment:
I realize only now, looking at [the docs for `req.acceptsLanguage()`](https://expressjs.com/en/api.html#req.acceptsLanguages), that it's not just a parsed header accessor but can be used to actually match accepted by supported languages by providing arguments:

```js
req.acceptsLanguages("en", "zh-cn", "de-ch", "es", "pt", "ru", …)
```

Seeing the switch this PR made from `req.acceptsLanguage()` to `req.headers['accept-language']` made me realize I might be missing something. @joverlee521 Were you aware of that? :-)

Reply:
Oh wow. completely missed that! 🤦‍♀